packages/flutter/lib/src/material/scrollbar.dart

```diff
@@ -96,9 +102,15 @@ class _ScrollbarState extends RawScrollbarState<Scrollbar> {
   bool _dragIsActive = false;
   bool _hoverIsActive = false;
   late ColorScheme _colorScheme;
+  late ScrollbarThemeData _scrollbarTheme;
```

Comment:
Caching the colorScheme and scrollbarTheme is OK, although it's not strictly necessary. Theme.of(context) is essentially just a single hashtable lookup.